package-lock.json

```diff
@@ -5,22 +5,22 @@
   "requires": true,
   "dependencies": {
     "@babel/runtime": {
-      "version": "7.2.0",
-      "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.2.0.tgz",
-      "integrity": "sha512-oouEibCbHMVdZSDlJBO6bZmID/zA/G/Qx3H1d3rSNPTD+L8UNKvCat7aKWSJ74zYbm5zWGh0GQN0hKj8zYFTCg==",
+      "version": "7.5.5",
```

Comment:
@zoldello After talking to @forresttanaka and @emro these package upgrades should be deferred and done as a separate ticket; please take them out if they are not dependencies of day.js

Reply:
@hitz Are you referring to packages in package-lock.json? If yes, it is updated by npm and it pull in a new package if a minor or patch update requires one. I recommend considering it a black box. Now, for package.json, I only added the needed package.

Reply:
I see optional:true was added. I believe it is harmless (https://stackoverflow.com/a/55373854/178550)